setup.cfg

```diff
@@ -31,25 +31,73 @@ exclude =
     env  # exclude asv benchmark environments from linting
 
 [flake8-rst]
-ignore =
-    F821,  # undefined name
-    W391,  # blank line at end of file [Seems to be a bug (v0.4.1)]
+ignore = E402,  # module level import not at top of file
+         W503,  # line break before binary operator
```

Comment:
aren't we adding the semicolon at the end then?

Reply:
ignoring `W503` allows for 
```
x = (a.....
       - b....)
``` 
So there may be a `\n` before a `-`, `+` ...

ignoring `W504`? would enforce
 ```
x = (a..... - 
       b....)
``` 
With a `\n` after the `-`, `+`, ...

Reply:
the question was if you wanted to ignore the errors for the semicolon in `df.plot();` in this PR.

Reply:
No, I'd rather have it on a per file based setting. It's only useful if there are many plots in one file that create cryptic outputs. If not it's better to have it checked.